Backend/scala-backend/app/models/caseClasses/StudentCC.scala

```diff
@@ -0,0 +1,9 @@
+package models
+
+final case class StudentCC(
```

Comment:
when frontend creates a student: id, groupBelonging are missing (i.e. not part of the json)